src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -155,6 +160,80 @@ private void OnGUI()
             // project = EditorGUILayout.TextField("Project", project);
             // EditorGUILayout.EndToggleGroup();
         }
+
+        /// <summary>
+        /// Creates Sentry folder for storing its configs - Assets/Resources/Sentry
+        /// </summary>
+        private static void TryCreateSentryFolder()
+        {
+            // TODO: revise, 'Resources' is a special Unity folder which is created by default. Not sure this check is needed.
+            if (!AssetDatabase.IsValidFolder("Assets/Resources"))
+            {
+                AssetDatabase.CreateFolder("Assets", "Resources");
+            }
+            if (!AssetDatabase.IsValidFolder($"Assets/Resources/{UnitySentryOptions.ConfigRootFolder}"))
+            {
+                AssetDatabase.CreateFolder("Assets/Resources", UnitySentryOptions.ConfigRootFolder);
+            }
+        }
+
+        /// <summary>
+        /// Find and copy 'link.xml' into current Unity project for IL2CPP builds
+        /// </summary>
+        private static void TryCopyLinkXml()
+        {
+            const string linkXmlFileName = "link.xml";
+
+            var linkXmlPath = $"{Application.dataPath}/Resources/{UnitySentryOptions.ConfigRootFolder}/{linkXmlFileName}";
+            if (File.Exists(linkXmlPath))
+            {
+                // TODO: should we log here?

```

Comment:
at the "debug" level it would be nice.
`else` could log in debug level: no link.xml exists yet, creating one.